proto/regen/ecocredit/v1/tx.proto

```diff
@@ -316,3 +333,65 @@ message MsgUpdateClassMetadata {
 
 // MsgUpdateClassMetadataResponse is the MsgUpdateClassMetadata response type.
 message MsgUpdateClassMetadataResponse {}
+
+// MsgAddCreditType is the Msg/AddCreditType request type.
+message MsgAddCreditType {
+
+  // root_address is the address of the signer.
+  // This MUST equal the address of the gov module for the tx to succeed.
+  string root_address = 1;
+
+  // credit_types defines a credit type to add to the credit types parameter.
+  repeated CreditType credit_types = 2;
+}
+
+// MsgAddCreditTypeResponse is the Msg/AddCreditType response type.
+message MsgAddCreditTypeResponse{}
+
+// MsgUpdateClassCreatorAllowlist is the Msg/UpdateClassCreatorAllowlist request type.
+message MsgUpdateClassCreatorAllowlist{
+
+  // root_address is the address of the signer.
+  // This MUST equal the address of the gov module for the tx to succeed.
+  string root_address = 1;
+
+  // add_creators is a list of addresses to add to the allowed class creator list
+  repeated string add_creators = 2;
+
+  // remove_creators is a list of addresses to remove from the allowed class creator list
+  repeated string remove_creators = 3;
+}
+
+// MsgUpdateClassCreatorAllowlistResponse is the Msg/UpdateClassCreatorAllowlist response type.
+message MsgUpdateClassCreatorAllowlistResponse {}
+
+// MsgToggleAllowlist is the Msg/ToggleAllowlist request type.
+message MsgToggleAllowlist {
+
+  // root_address is the address of the signer.
+  // This MUST equal the address of the gov module for the tx to succeed.
+  string root_address = 1;
+
+  // setting defines the boolean value to set the allowlist on or off.
+  bool setting = 2;
+}
+
+// MsgToggleAllowlistResponse is the Msg/ToggleAllowlist response type.
+message MsgToggleAllowlistResponse {}
+
+// MsgUpdateClassFee is the Msg/UpdateClassFee request type.
+message MsgUpdateClassFee {
+
+  // root_address is the address of the signer.
+  // This MUST equal the address of the gov module for the tx to succeed.
+  string root_address = 1;
+
+  // add_fees defines a list of coins to append to the list of fees that can be used in the class creation fee.
+  repeated cosmos.base.v1beta1.Coin add_fees = 2;
+
+  // remove_fee_denoms defines a list of denoms to remove from the list of class creation fees.
+  repeated string remove_fee_denoms = 3;
+}
```

Comment:
As mentioned above, I think we want to support a single credit class fee and this should update that single fee.